code/modules/mob/living/carbon/martian/martian.dm

```diff
@@ -128,3 +130,19 @@
 		stat = CONSCIOUS
 	else
 		health = maxHealth - getOxyLoss() - getFireLoss() - getBruteLoss() - getCloneLoss()
+
+/mob/living/carbon/martian/Stat()
```

Comment:
This does not work, and I'm not too sure why.

They just do not get the stat tab